lib/puppet/transaction/event_manager.rb

```diff
@@ -147,19 +147,19 @@ def process_callback(resource, callback, events)
     resource.send(callback)
 
     if not resource.is_a?(Puppet::Type.type(:whit))
-      resource.notice "Triggered '#{callback}' from #{events.length} events"
+      resource.notice _("Triggered '#{callback}' from #{events.length} events")
```

Comment:
I think for externalizing strings with pluralization, we should take a different form. In this case, I think it looks something like:

```ruby
resource.notice n_("Triggered '#{callback}' from %{count} event", "Triggered '#{callback}' from %{count} events", events.length) % { :count => events.length }
```

though strictly speaking that would change this string to include `event` singular or `events` plural even in English depending on the length of the `events` array... but I think thats ok given we're pushing this towards pup 5